From 66521f64c20ff30a5d96ad505e39641fd812a167 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Karel=20Ko=C4=8D=C3=AD?= <cynerd@email.cz>
Date: Fri, 24 Jan 2020 13:48:40 +0100
Subject: [PATCH] hostapd: fix space variable and configuration that way
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

The hack used to get space variable probably no longer works with newest
make. This replaces it with version suggested in GNU make documentation.
http://www.gnu.org/software/make/manual/make.html#Flavors

Signed-off-by: Karel Kočí <cynerd@email.cz>
---
 package/network/services/hostapd/Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/package/network/services/hostapd/Makefile b/package/network/services/hostapd/Makefile
index 9712e72..bba0654 100644
--- a/package/network/services/hostapd/Makefile
+++ b/package/network/services/hostapd/Makefile
@@ -88,8 +88,8 @@ DRIVER_MAKEOPTS= \
 	CONFIG_IEEE80211AC=$(HOSTAPD_IEEE80211AC) \
 	CONFIG_DRIVER_WEXT=$(CONFIG_DRIVER_WEXT_SUPPORT) \
 
-space :=
-space +=
+nullstring :=
+space := $(nullstring) # end of the line
 
 ifeq ($(LOCAL_VARIANT),full)
   DRIVER_MAKEOPTS += CONFIG_IEEE80211W=$(CONFIG_DRIVER_11W_SUPPORT)
-- 
2.25.0

